### PR TITLE
fix(docs): Fix images path in docs

### PR DESCRIPTION
**Pull request by nidzix**

### CODE REVIEW

**Comment by ioanpocol** [comment]
:+1:

File: client/tasks/options/copy.js [hunk 1]
```diff
@@ -13,17 +13,28 @@ module.exports = {
             ]
         }]
     },
-    docs: {
+    docsViews: {
```
Comment (petrjasek) [inline client/tasks/options/copy.js]: no need to split, `files` is list

Comment (nidzix) [inline client/tasks/options/copy.js]: :+1:

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**